CameraXBasic/app/src/main/AndroidManifest.xml

```diff
@@ -30,8 +30,12 @@
     <uses-permission android:name="android.permission.CAMERA" />
 
     <application
+        android:icon="@mipmap/ic_launcher"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:name=".CameraXBasic"
         android:label="@string/app_name"
-        android:allowBackup="true">
+        tools:ignore="AllowBackup">
```

Comment:
Because it's an example application, so I thought it's not required (the previous version had lots of other code-quality fixes, which I've added back in, just alike those excessive indents with 8 spaces). Can revert that that.

Reply:
As a sample application, we try to implement best practices by default. Rather than explicitly ignoring it, it's much easier to simply allow for the application to be backed up. I see no good reason for this to be changed.

Reply:
I often just do what the code-inspection tells me to do. Will keep that in mind.